docs/README.rst

```diff
@@ -0,0 +1,157 @@
+:ref:`intro_testers`
+
+
+Provided Utilities
+==================
+* ``teuthology`` - Run individual jobs
+* ``teuthology-coverage`` - Analyze code coverage via lcov
+* ``teuthology-kill`` - Kill running jobs or entire runs
+* ``teuthology-lock`` - Lock, unlock, and update status of machines
+* ``teuthology-ls`` - List job results by examining an archive directory
+* ``teuthology-openstack`` - Use OpenStack backend (wrapper around ``teuthology-suite``)
+* ``teuthology-nuke`` - Attempt to return a machine to a pristine state
+* ``teuthology-queue`` - List, or delete, jobs in the queue
+* ``teuthology-report`` - Submit test results to a web service (we use `paddles <https://github.com/ceph/paddles/>`__)
+* ``teuthology-results`` - Examing a finished run and email results
+* ``teuthology-schedule`` - Schedule a single job
+* ``teuthology-suite`` - Schedule a full run based on a suite (see `suites` in `ceph-qa-suite <https://github.com/ceph/ceph-qa-suite>`__)
+* ``teuthology-updatekeys`` - Update SSH host keys for a mchine
+* ``teuthology-worker`` - Worker daemon to monitor the queue and execute jobs
+
+For a description of the distinct services that utilities interact with see
+:ref:`components`.
+
+Installation
+============
+
+See :ref:`installation_and_setup`.
+
+
+Infrastructure
+==============
+
+The examples in this document are based on the lab machine configuration used
+by the Red Hat Ceph development and quality assurance teams 
+(see :ref:`lab_setup`). Other instances of a Ceph Lab being used in a
+development or testing environment may differ from these examples.
+
+
+Detailed test configuration
+===========================
+
+See :ref:`detailed_test_config`.
+
+
+Virtual Machine Support
+=======================
+
+For OpenStack support, see :ref:`openstack-backend`
+
+For 'vps' support using `downburst <https://github.com/ceph/downburst>`__, see
+:ref:`downburst_vms`
+
+
+Test Suites
+===========
+
+Most of the current teuthology test suite execution scripts automatically
+download their tests from the master branch of the appropriate github
+repository.  People who want to run experimental test suites usually modify the
+download method in the ``teuthology/task`` script to use some other branch or
+repository. This should be generalized in later teuthology releases.
+Teuthology QA suites can be found in ``src/ceph-qa-suite``. Make sure that this
+directory exists in your source tree before running the test suites.
+
+Each suite name is determined by the name of the directory in ``ceph-qa-suite``
+that contains that suite. The directory contains subdirectories and yaml files,
+which, when assembled, produce valid tests that can be run. The test suite
+application generates combinations of these files and thus ends up running a
+set of tests based off the data in the directory for the suite.
+
+To run a suite, enter::
+
+    teuthology-suite -s <suite> [-c <ceph>] [-k <kernel>] [-e email] [-f flavor] [-t <teuth>] [-m <mtype>]
+
```

Comment:
To be honest, I thought that this was a field that I saw in the docs but I can't find it now.  Please ignore for now.